lib/cmd/fh3/projects.js

```diff
@@ -52,6 +52,10 @@ function projects(argv, cb) {
 function listProjects(args, cb) {
   common.listProjects(function (err, projs) {
     if (err) return cb(err);
+    //Allow filtering by author email
+    if(args[1]){
```

Comment:
@wtrocki I'm not using an option here. (Like --env) its just an optional arg at the end of the command.